app/src/main/java/org/fossasia/openevent/general/notification/NotificationService.kt

```diff
@@ -23,4 +24,12 @@ class NotificationService(
             it
         }
     }
+
+    fun updateNotification(notification: Notification): Single<Notification> {
+        return notificationApi.updateNotification(notification.id, notification).map {
+            Timber.e(notification.isRead.toString() + "\n" + it.isRead.toString())
```

Comment:
I need help here. 
I'm getting - 
```
E/NotificationService$updateNotification: true
    false
```

with API - 
```
@PATCH("notifications/{notification_id}")
    fun updateNotification(
        @Path("notification_id") notificationId: Long,
        @Body notification: Notification
    ): Single<Notification>
```
https://api.eventyay.com/#notifications-notification-detail-patch

Is it a server side issue?

Reply:
@iamareebjamal Notification is not updating with string also, can you please check my implementation so that I can open up an issue on the server.

Reply:
Open the issue

Reply:
> Open the issue

https://github.com/fossasia/open-event-server/issues/6104#issuecomment-505838496